src/Apps/Sell/Components/StickyNavigation.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components"
+import { Flex, Button } from "@artsy/palette"
+import { SubmissionContext } from "Apps/Sell/SubmissionContext"
+import { themeGet } from "@styled-system/theme-get"
+import { useContext } from "react"
+
+export const StickyNavigation = () => {
+  const { actions: { isDirty, goBack, isSaving, submitSubmission, saveAndContinue, currentStep, isValid, atLastStep, atFirstStep } } = useContext(SubmissionContext)
+
+  if (currentStep() === "thank-you") return null
+
+  return (
+    <StickyContainer>
+      <Flex
+        m={2}
+        p={2}
+        width="100%"
+        flexDirection="row"
+        justifyContent="space-between"
+      >
+        <Button
+          disabled={atFirstStep()}
+          variant="secondaryBlack"
+          onClick={goBack}
+        >
+          Back
+        </Button>
+        {atLastStep() ? (
+          <Button
+            variant="primaryBlack"
+            onClick={submitSubmission}
+          >
+            Submit
+          </Button>
+        ) : (
+          <Button
+            variant={isDirty() ? "primaryBlack" : "secondaryBlack"}
+            disabled={!isValid()}
+            loading={isSaving()}
+            onClick={saveAndContinue}
+          >
+            {isDirty() ? "Continue" : "Skip" }
+          </Button>
+        )}
+      </Flex>
+    </StickyContainer>
+  )
+}
+
+const StickyContainer = styled(Flex)`
```

Comment:
Is that component used for the same purpose? Upon reading, it seemed like that was meant for scrolling behavior where you want a component to stick rather than float up with the rest of the scrolling content. I suppose this is more of a fixed placement, rather than sticky in that sense.

Reply:
I have no idea what I'm looking at here but...

If you want something to be position fixed then you need to accommodate the rest of the UI; which the `Sticky` component does. We're going to be moving the entire global nav into this as well, which is the last hold out.

The manual height calc below is going to be very brittle.

Reply:
You can achieve this layout without using anything `fixed` (or sticky for that matter) actually.

Reply:
I was thinking more about steps like [Upload photos ](https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw/Submission-%26-Pre-Consignment-Management?node-id=1448-6788&t=Esj7CSStlPd6lkW0-4) where we will definitely have a scroll if the user uploads more photos, could prevent future issues.

Reply:
The decision was to have a sticky footer and scroll the rest of the page when there is too much content - it will be addressed separately in another ticket.

Reply:
I used the `<Sticky>` component for the header but I didn't seem to be a good fit for the fixed footer. Open to suggestions or if there's another reference I should look at.

Reply:
One could just write a normal flex layout — no need for `fixed`; just `overflow: auto` where appropriate?

Reply:
Yes, we can go with something more simple if the Sticky didn't work, without height calculations and fixed position. I can look into it also in the other ticket once we have more content, if other comments are already addressed and ready to be approved.

Reply:
Just paired on this with @olerichter00 to provide a Flex layout which you'll find in the last commit 👍🏼